src/components/ProgressIndicator/ProgressIndicator.stories.tsx

```diff
@@ -147,6 +155,76 @@ export const Labels: Story = {
   ),
 };
 
+export const LabelPlacements: Story = {
+  render: () => (
+    <SimpleGrid columns={1} gap="grid.m">
+      <HStack spacing="s">
```

Comment:
Similarly, to help with visual clarity, bump the spacing value to `"l"` for the `<HStack>` components.